components/support/base/src/main/java/mozilla/components/support/base/ids/SharedIdsHelper.kt

```diff
@@ -4,8 +4,11 @@
 
 package mozilla.components.support.base.ids
```

Comment:
Should these go under `support-ktx` instead? It already has `support-base` as a dependency so I don't think we'd be adding overhead there.

Reply:
Sure, I'll move it.

Reply:
There is one issue when I move it, it will still break Fenix.

Reply:
Oh, they're using these extensions from a-c already? 🤔

Reply:
That is to say, wouldn't they have to import the extension from this package to use it?

Reply:
Yes, in `PoweredByNotification.kt`.  Another choice is I don't add these extensions but remove the call in Fenix?

Reply:
Yes, they are importing `mozilla.components.support.base.ids.cancel` and `mozilla.components.support.base.ids.notify`

Reply:
Spoke offline: these extensions existed before, and we're reintroducing them since it was breaking Fenix. We can refactor them to the right place later then. 👍